tests/models/test_ngsi_ld_entities.py

```diff
@@ -0,0 +1,533 @@
+import _json
+import unittest
+from pydantic import ValidationError
+#from filip.clients.ngsi_v2.cb import ContextBrokerClient	
+
+from filip.clients.ngsi_ld.cb import ContextBrokerLDClient
+# from filip.models.ngsi_v2.subscriptions import \
+#     Http, \
+#     HttpCustom, \
+#     Mqtt, \
+#     MqttCustom, \
+#     Notification, \
+#     Subscription
+from filip.models.base import FiwareLDHeader
+from filip.utils.cleanup import clear_all, clean_test
+from tests.config import settings
+from filip.models.ngsi_ld.context import \
+    ContextLDEntity, \
+    ContextProperty, \
+    ContextRelationship, \
+    NamedContextProperty, \
+    ActionTypeLD
+import requests
+
+class TestEntities(unittest.TestCase):
+    """
+    Test class for entity endpoints.
+    """
+
+    def setUp(self) -> None:
+        """
+        Setup test data
+        Returns:
+            None
+        """
+        # self.fiware_header = FiwareLDHeader(
+        #     service=settings.FIWARE_SERVICE,
+        #     service_path=settings.FIWARE_SERVICEPATH)
+        self.fiware_header = FiwareLDHeader()
+        self.http_url = "https://test.de:80"
+        self.mqtt_url = "mqtt://test.de:1883"
+        self.mqtt_topic = '/filip/testing'
+
+        #CB_URL = "http://localhost:1026"
+        CB_URL = "http://137.226.248.200:1027"
+        self.cb_client = ContextBrokerLDClient(url=CB_URL,
+                                    fiware_header=self.fiware_header)
+        
```

Comment:
Add a cleanup step here will increase the robustness of the tests

Reply:
fixed